javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,66 @@
+/**freeCodeCamp's username qamarfj */
+/**Age-ify (A future age calculator) */
+const yearOfBirth = 1978; // can't be change
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log('You will be ' + age + ' years old in ' + yearFuture + '.');
+
+/* Goodboy-Oldboy (A dog age calculator)*/
+const dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+const dogYear = 7;
+let shouldShowResultInDogYears = true;//Boolean variable 
+if (shouldShowResultInDogYears) {
+    console.log('Your dog will be ' + (dogYearFuture - dogYearOfBirth) +
+        ' human years old in ' + yearFuture + '.');
+}
+
+else
+    console.log('Your dog will be ' + (dogYearFuture - dogYearOfBirth) *
+        dogYear + ' dog years old in ' + yearFuture + '.');
+
+/*Housey pricey (A house price estimator)*/
+let width;
+let depth;
+let height;
+let gardenSizeInM2;
+let houseCost;
+let housePrice;
+let estimat;
+function estimator() {
+    let volumeInMeters = width * depth * height;
+    housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+    if (houseCost >= housePrice) return " too much";
+    else return " too little";
```

Comment:
What if the cost was just right? Do you think you should use a third string to represent that?

Reply:
Thank you for  the feedback on my homework.
I have added check for this option.

Reply:
You're welcome :)